File: C/EX17.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char const *argv[])
{
    int A, B, C, hip, soma;

    printf("Digite o valor para: \nA:");
    scanf("%d", &A);

    printf("B:");
    scanf("%d", &B);

    printf("C:");
    scanf("%d", &C);

    hip = A * A;
    soma = (B*B) + (C*C);

    if (hip == soma)
    {
        printf("Triângulo retângulo");
    }

    else
    {
        hip = B * B;
        soma = (A*A) + (C*C);

        if (hip == soma)
        {
            printf("Triângulo retângulo");
        }

        else
        {
            hip = C * C;
            soma = (A*A) + (B*B);
            
            if (hip == soma)    
            {
                printf("Triângulo retângulo");
            }
            
            else
            {
                printf("Não forma um triângulo retângulo");
            }
            
            
        }
        
        
    }
    
    
        
    return 0;
}

File: Python/EX17.py
# verifica se forma um triângulo retângulo

print("Informe os lados do triângulo.")

hip = int(input("Hip: "))
c1 = int(input("cat: "))
c2 = int(input("cat: "))

hip = hip**2

soma = (c1**2) + (c2**2)

if hip == soma:
    print("Triangulo retângulo")
else:
    print("Não é um triangulo retângulo")